app/lib/server/functions/getFullUserData.js

```diff
@@ -30,6 +30,7 @@ const fullFields = {
 	requirePasswordChange: 1,
 	requirePasswordChangeReason: 1,
 	roles: 1,
+	extension: 1,
```

Comment:
I added this for now, but, if we don't want to leak this prop the client, maybe we should remove it, wdyt?

Reply:
If the client wants to login to SIP, it is going to query this property. So it is ok if we pass it on.